sunpy/timeseries/sources/eve.py

```diff
@@ -69,23 +85,12 @@ def peek(self, title='EVE/ESP Level1', **kwargs):
         """
 
         self._validate_data_for_plotting()
```

Comment:
Does this not belong in plot?

Reply:
Should this be moved there? It'd make sense I suppose.

Reply:
Well why would we only want to validate the data for peek but not plot?

Reply:
Yeah that makes sense, I'll make that change.